src/components/shared/Repository/TagsList.tsx

```diff
@@ -39,6 +37,24 @@ export default function TagsList(props) {
   // TODO: refactor
   const [organization, ...repo] = location.pathname.split('/').slice(2);
   const repository = repo.join('/');
+  const PageBreadcrumbs = [
+    {
+      title: 'organizations',
+      id: 'namespace-breadcrumb',
+      to: '#',
```

Comment:
shouldn't these point to something? Like this should go to the org list instead of `#`

Reply:
Hmm, yes. I will change it.